packages/plugins/src/namespaced/index.ts

```diff
@@ -103,7 +103,6 @@ export const Combobox = {
   Group: ComboboxGroup,
   Label: ComboboxLabel,
   Content: ComboboxContent,
-  Empty: ComboboxEmpty,
```

Comment:
@zernonia can i ask you why it was removed?

Reply:
Might be a mistake when running `generate` script. Thanks for noticing!